src/core/nsfs.js

```diff
@@ -143,6 +109,43 @@ async function main(argv = minimist(process.argv.slice(2))) {
     }
 }
 
+function make_object_sdk_instance(fs_root, fs_config) {
```

Comment:
how about defining this to be `function init_request_sdk(req, res, fs_root, fs_config) {...}`